gvr-renderableview/app/src/main/res/layout/activity_main.xml

```diff
@@ -2,8 +2,9 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:background="@color/white_transparent"
```

Comment:
I get an aapt error: ``AAPT: error: resource color/white_transparent (aka org.gearvrf.renderableview:color/white_transparent) not found.``